streaming/streaming.go

```diff
@@ -169,6 +171,7 @@ func getMysqlSourceStreamConfig(dbList []profiles.LogicalShard) *datastreampb.So
 		}
 		includeDbList = append(includeDbList, includeDb)
 	}
+	fmt.Printf("Include DB List for datastream: %+v\n", includeDbList)
```

Comment:
nit: move the fmt.printf to logs

Reply:
Initialising the logger is tricky sometimes due to the way its been integrated. I want to hold off for logs replacement and do it in a dedicated log cleanup task.
I've added a TODO here instead for now, hope that works.